builtin/settingtypes.txt

```diff
@@ -484,10 +477,10 @@ pause_fps_max (FPS in pause menu) int 20
 viewing_range (Viewing range) int 100 20 4000
 
 #    Width component of the initial window size.
-screen_w (Screen width) int 800
+screenW (Screen width) int 800
 
 #    Height component of the initial window size.
-screen_h (Screen height) int 600
+screenH (Screen height) int 600
```

Comment:
Same problem here. The PR is good when there are no copy&paste "revert" leftovers from outdated files. There's the `git rebase` function if you want to update your branch while keeping your commits.